Gems/Atom/Feature/Common/Code/Source/Material/MaterialConverterSystemComponent.cpp

```diff
@@ -183,6 +187,13 @@ namespace AZ
                 sourceData.SetPropertyValue(Name{"roughness.factor"}, roughness.value());
             }
 
+            if (!materialData.GetRoughnessFactor().has_value() && abs(materialData.GetShininess()) > AZ::Constants::FloatEpsilon)
+            {
+                // When the MaterialData provides Shininess instead of Roughness, it is necessary to convert Shininess to Roughness.
+                sourceData.SetPropertyValue(
+                    Name{ "roughness.factor" }, aznumeric_cast<float>(pow(2 / (materialData.GetShininess() + 2), 0.25)));
```

Comment:
Can you add a comment explaining why this is the correct conversion from shininess to roughness?

nitpick: should probably use 2.0 instead of 2 since they are not integers here.

Reply:
there is no clear cut interpretation/conversion for this. 
Normalized Blinn-Phong: D_p(m)=((alpha_p+2)/(2*PI))dot(n,m)^alpha_p
Usually use formula (UE4 use this): alpha_p=2*alpha^-2 - 2, alpha=roughness^2